modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1735,10 +1735,19 @@ protected int getCellIndex(T cell){
      *                                                                         *
      **************************************************************************/
 
-    final VirtualScrollBar getHbar() {
+    /**
+     *
+     * @return The scroll bar used for scrolling horizontally.
+     */
+    protected final ScrollBar getHbar() {
         return hbar;
     }
-    final VirtualScrollBar getVbar() {
+
+    /**
+     *
+     * @return The scroll bar used for scrolling vertically.
+     */
+    protected final ScrollBar getVbar() {
```

Comment:
getHbar() and gVbar() should be made protected because anyone overriding the VirtualFlow will need to control them. I have several examples to support that:
- I have implemented a zoom property in the SpreadhsheetView that shrink or enhance the Grid. And as I am doing that, I still want the ScrollBar to have the same size (like Excel). Therefore, I must compensate the zoom factor applied on the grid, on the ScrollBar.
- I have added a Rectangle object on my Grid to provide a visual black Rectangle indicating the selected cells (just like Excel). This Rectangle is added on top of everything and I need to clip it if I don't want it to overlap the scrollBars. Therefore, I need to have access to the scrollBars and especially their width and height property in this case.